NAMESPACE

```diff
@@ -2,16 +2,23 @@
 
 export(convert_docx_to_rmd)
 export(create_sfp)
+export(get_path_to_protocol)
 export(get_protocolnumbers)
 export(get_short_titles)
+export(render_all)
```

Comment:
Only `get_path_to_protocol` should be internal. The other `get_` functions are useful for the users too and I refer to them in the help of `create_sfp`. 
Regarding `render_all()` it will need to be changed in order to make it do what is said in the discussion in #18. I think about making that an internal function as well.

Reply:
Perfect.